### PR TITLE
prune tests

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?
- minor formating docstrings in tests, 
- optimize imports and 
- dropping unused variables in `tests.utils`

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

### CODE REVIEW

File: tests/test_cpu_models.py [hunk 1]
```diff
@@ -84,7 +79,7 @@ def test_default_logger_callbacks_cpu_model():
     )
 
     model, hparams = tutils.get_model()
-    tutils.run_model_test_no_loggers(trainer_options, model, hparams, on_gpu=False)
+    tutils.run_model_test_no_loggers(trainer_options, model)
```
Comment (williamFalcon) [inline tests/test_cpu_models.py]: on_gpu=False is missing?

Comment (Borda) [reply tests/test_cpu_models.py]: it was not used inside the function...

**Comment by williamFalcon** [comment]
@Borda rebase?

Also, the diff shows the gpu argument is being changed

**Comment by Borda** [comment]
@williamFalcon rebased :]

**Pull request merged**